apps/give-cast/utils/receipt.ts

```
@@ -0,0 +1,13 @@
+import { postApi } from '@/utils/api';
+
+export default async function sendReceipt(data: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)